tools/polarshift.rb

```diff
@@ -469,9 +469,13 @@ def transform_custom_fields(src)
     def clone_run(test_run_id: nil)
       template_hash = {}
       query_result = polarshift.get_run_smart(project, test_run_id)
-
       tc_ids = extract_test_case_ids(query_result)
-      custom_fields = transform_custom_fields(query_result['customFields']['Custom'])
+      if query_result['customFields'].nil?
+        # hard-code it to something
+        custom_fields = { "caseimportance" => "critical", "description" => "" }
```

Comment:
Hard code case importance to critical is expected ?

Reply:
@liangxia it just need something there.  I just took that from a working example.  Don't know what are the available options

Reply:
@pruan-rht , how about defaulting to an empty hash?